apps/hasura-nextjs/.env.example

```diff
@@ -0,0 +1,9 @@
+NEXTAUTH_URL=http://localhost:3000
```

Comment:
This file can be removed since we have `.env` checked in

Reply:
Done